modules/ROOT/pages/servers-create.adoc

```diff
@@ -16,6 +16,10 @@ To add a Mule server to the Runtime Manager console so that you can manage it, y
 
 * Your enterprise license is current. 
 * You are running Mule 3.6.0 or later, and API gateway 2.1 or later.
++
+For Mule 3.6.x, install the Runtime Manager agent.
++
+For information, see xref:installing-and-configuring-runtime-manager-agent.adoc[Install or Update the Runtime Manager Agent].
```

Comment:
Does your audience REALLY need this xref? 
We want to start putting nice-to-haves in a ==See Also topic and only need-to-haves in the running narrative, to avoid sending readers off somewhere else.
Also, if we do send them off, we're to tell them EXACTLY why, and not just "for more information."

Reply:
Leaving as it is a prereq.